lib/routes/lnurl/widgets/lnurl_metadata.dart

```diff
@@ -13,14 +13,18 @@ class LNURLMetadata extends StatelessWidget {
         metadataMap['image/png;base64'] ?? metadataMap['image/jpeg;base64'];
     return Column(
       mainAxisSize: MainAxisSize.max,
-      crossAxisAlignment: CrossAxisAlignment.center,
+      crossAxisAlignment: CrossAxisAlignment.stretch,
       children: [
         Text(
           metadataMap['text/long-desc'] ?? metadataMap['text/plain'],
-          style: Theme.of(context).textTheme.bodySmall,
+          style: Theme.of(context).textTheme.bodyLarge,
         ),
-        _LNURLMetadataImage(
-          base64String: base64String,
+        Container(
+          alignment: Alignment.center,
+          padding: const EdgeInsets.symmetric(vertical: 150),
```

Comment:
Remove the padding here, or use a much acceptable value in the range of 8-24 and wrap the Container with an Expanded widget so it's centered in the available space.
There are lnurl payments where payee details need to be filled and this wouldn't look great on that case, you can see for yourself by reversing the conditions to display those widgets.

Reply:
Ran into some difficulties when trying wrap the container with an expanded widget. This is most likely due to the constrains being unbounded somewhere